src/components/pages/Signup/Signup.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+// import { reach } from 'yup'
+// import schema from '../../../utils/validation/formSchema'
+import { Form, Input, Button, Radio } from 'antd';
+
+const initialFormValues = {
+  email: '',
+  password: '',
+  firstName: '',
+  lastName: '',
+};
+
+const initialFormErrors = {
+  email: '',
+  password: '',
+  firstName: '',
+  lastName: '',
+};
+
+const Signup = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  const [formValues, setFormValues] = useState(initialFormValues);
+  const [formErrors, setFormErrors] = useState(initialFormErrors);
+
+  const [value, setValue] = React.useState(1);
+
+  const postNewAccount = async newAccount => {
+    try {
+      const response = await axios.post(' ', newAccount);
+      console.log(response);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  // const validate = (name, value) => {
+
+  //     reach(schema, name)
+  //         .validate(value)
+  //         .then(()=> setFormErrors({...formErrors, [name]: '' }))
+  //         .catch(err => setFormErrors({...formErrors, [name]: err.errors[0]}))
+  // }
+
+  const formSubmit = () => {
+    const newAccount = {
+      username: formValues.username.trim(),
+      password: formValues.password.trim(),
+      firstName: formValues.firstName.trim(),
+      lastName: formValues.lastName.trim(),
+    };
+    console.log(newAccount);
+    postNewAccount(newAccount);
+  };
+
+  const onChange = e => {
+    console.log('radio checked', e.target.value);
+    setValue(e.target.value);
+  };
```

Comment:
I'm assuming this is meant to be a placeholder until we connect with the back-end for asserting these roles?